fast/stages/0-bootstrap/organization.tf

```diff
@@ -130,9 +130,17 @@ import {
   to = module.organization.google_org_policy_policy.default[each.key]
 }
 
-module "organization" {
+module "organization-logging" {
```

Comment:
Can you provide a comment here, why we are invoking `modules/organization` twice?

Reply:
Adding the `logging_settings` to the main organization block results in a circular dependency

>Error: Cycle: module.log-export-logbucket.google_logging_linked_dataset.dataset, module.log-export-logbucket (close), module.log-export-logbucket.google_logging_billing_account_bucket_config.bucket, module.log-export-logbucket.google_logging_organization_bucket_config.bucket, module.log-export-logbucket.google_logging_folder_bucket_config.bucket, module.log-export-logbucket.var.kms_key_name (expand), module.log-export-logbucket.var.location (expand), module.log-export-logbucket.var.id (expand), module.log-export-logbucket.var.retention (expand), module.log-export-logbucket.var.parent_type (expand), module.log-export-logbucket.var.parent (expand), module.log-export-logbucket.var.log_analytics (expand), module.log-export-logbucket.var.description (expand), module.log-export-logbucket.google_logging_project_bucket_config.bucket, module.organization.google_project_iam_member.bucket-sinks-binding, module.organization.google_storage_bucket_iam_member.storage-sinks-binding, module.organization.google_project_iam_member.project-sinks-binding, module.organization.google_pubsub_topic_iam_member.pubsub-sinks-binding, module.organization.local.sink_bindings (expand), module.organization.google_bigquery_dataset_iam_member.bq-sinks-binding, module.organization (close), module.log-export-logbucket (expand), module.log-export-logbucket.output.id (expand), local.log_sink_destinations (expand), module.organization.var.logging_sinks (expand), module.organization.local.logging_sinks (expand), module.organization.google_logging_organization_sink.sink, module.organization.output.sink_writer_identities (expand)

Reply:
👍 it's a fairly common pattern in FAST: there are circular dependencies in org/folder which are sometimes inevitable and due to logical issues, Wiktor we had a look at a similar thing a while ago when you tried to change the org (or folder?) module output dependencies

Reply:
I understand why it is here. I just think, that it is clear when I look it at the PR, but when I'll look at the code it might be not so obvious, so I just asked to have a comment in the code there for my future self :-)

Reply:
100% agree :)

Reply:
I added a comment [here](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/2139/files#diff-3ea77693c59807c5a29011b1021f810ebaee4fe591c7b565c493d55e526f0dfdR96). But you're right, le me add something here too.